app/components/ui/set-up-domain/progress-header/index.js

```diff
@@ -0,0 +1,42 @@
+// External dependencies
+import { bindHandlers } from 'react-bind-handlers';
+import i18n from 'i18n-calypso';
+import React, { PropTypes } from 'react';
+import withStyles from 'isomorphic-style-loader/lib/withStyles';
+
+// Internal dependencies
+import ProgressBar from 'components/ui/progress-bar';
+import styles from './styles.scss';
+
+class ProgressHeader extends React.Component {
```

Comment:
Failed to find functional components when I looked. Or at least most of the stuff under `set-up-domain` could've been like that, but wasn't, so I decided not to do it.